dnsapi/dns_1984hosting.sh

```diff
@@ -168,7 +168,7 @@ _1984hosting_login() {
   _debug2 response "$response"
 
   if [ "$response" = '{"loggedin": true, "ok": true}' ]; then
-    One984HOSTING_COOKIE="$(grep '^Set-Cookie:' "$HTTP_HEADER" | _tail_n 1 | _egrep_o 'sessionid=[^;]*;' | tr -d ';')"
+    One984HOSTING_COOKIE="$(grep '^set-cookie:' "$HTTP_HEADER" | _tail_n 1 | _egrep_o 'sessionid=[^;]*;' | tr -d ';')"
```

Comment:
why not use `"grep -i "` option?

Reply:
I was assuming this was a permanent change in how HTTP headers behaved and didn't think much of it. After looking at it more deeply though, using `grep -i` is probably more appropriate due to the differences between HTTP/1.1 and HTTP/2. I've been getting HTTP/2 headers where lower case header fields are the default, but HTTP/1.1 is case insensitive. Any combination of upper/lower case is valid for HTTP/1.1, but could be invalid for HTTP/2. Using `grep '^SET-COOKIE:\|Set-Cookie:\|Set-cookie:\|set-cookie:'` could probably be fine for more sane looking header fields. It would just break again if it were anything else though.

Reply:
I went ahead and took care of it.